### PR TITLE
ARC-2604 remove ff for deployment webhook and backfill

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
Remove ff
use-dynamodb-for-deployment-backfill
and
use-dynamodb-for-deployment-webhook

**Why**
Rollout to 100% already.

**Added feature flags**

**Affected issues**  
[ARC-2604]

**How has this been tested?**  
Unit test.

**Whats Next?**
Archive the ff

[ARC-2604]: https://softwareteams.atlassian.net/browse/ARC-2604

### CODE REVIEW

File: src/sync/deployment.test.ts [hunk 1]
```diff
@@ -361,18 +268,9 @@ describe("sync/deployments", () => {
...
-					variables: {
-						owner: "integrations",
-						repo: "test-repo-name",
-						per_page: 20
-					}
-				})
-				.query(true)
-				.reply(200, deploymentNodesFixture);
+			createGitHubNock(deploymentNodesFixture);
+			const lastEdges = deploymentNodesFixture.data.repository.deployments.edges;
```
Comment (gxueatlassian) [inline src/sync/deployment.test.ts:272]: Because the ff will coz polling the graphql twice

File: src/sync/deployment.test.ts [hunk 2]
```diff
@@ -407,6 +305,8 @@ describe("sync/deployments", () => {
 			githubUserTokenNock(installationId);
 
 			createGitHubNock(deploymentNodesFixture);
+			const lastEdges = deploymentNodesFixture.data.repository.deployments.edges;
```
Comment (gxueatlassian) [inline src/sync/deployment.test.ts:308]: Because the ff will coz polling the graphql twice

File: src/sync/deployment.test.ts [hunk 3]
```diff
@@ -482,14 +382,14 @@ describe("sync/deployments", () => {
 				"deploymentSequenceNumber": 500226426,
 				"updateSequenceNumber": 500226426,
 				"displayName": "[TEST-123] test-commit-message",
-				"url": null,
+				"url": "random-success",
```
Comment (gxueatlassian) [inline src/sync/deployment.test.ts:385]: Previous test didn't test the ff on scenario, how with ff on, this value will be extracted correctly.

File: src/transforms/transform-deployment.test.ts [hunk 4]
```diff
@@ -545,91 +541,6 @@ describe("transform GitHub webhook payload to Jira payload", () => {
 			]));
 		});
 
-		it(`supports branch and merge workflows, sending related commits in deploymentfor Cloud`, async () => {
```
Comment (gxueatlassian) [inline src/transforms/transform-deployment.test.ts:548]: This is same as above for ff off. So delete it.

**Review by yvonnevictoria — DISMISSED** [review]

**Review by yvonnevictoria — APPROVED** [review]

**Pull request merged**